src/Microsoft.AspNetCore.ResponseCaching/Internal/StringBuilderExtensions.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Text;
+
+namespace Microsoft.AspNetCore.ResponseCaching.Internal
+{
+    internal static class StringBuilderExtensions
```

Comment:
Hmm is this repo a good place to put this? It seems very generic and could be used elsewhere.